content/devs.json

```diff
@@ -303,5 +303,31 @@
         "url": "mailto:rhaidzsal.ali@gmail.com"
       }
     ]
+  },
+  {
+    "avatar": "https://avatars1.githubusercontent.com/u/36836488?s=460&u=3bad90fad66e85a12ff90883ea54e2d3ba8f7fe7&v=4",
+    "name": "Mark Dino Pelonia",
+    "title": "Junior Software Engineer",
+    "company": "Damstra Technology/Shore360",
+    "blurb": "Hi, I'm a JavaScript & React enthusiast, have an eye for design, convert mock design to a live website, and naturally curious.",
+    "skills": ["JavaScript", "React", "Gatsby", "NodeJS", "MongoDB", "GraphQL", "Sass", "Styled-components", "Git"],
```

Comment:
```
Mark Dino Pelonia has invalid details on devs.json
[ { message:
     '"skills[7]" length must be less than or equal to 15 characters long',
    path: [ 'skills', 7 ],
    type: 'string.max',
    context:
     { limit: 15,
       value: 'Styled-components',
       encoding: undefined,
       label: 'skills[7]',
       key: 7 } } ]
```

Reply:
Fixed! CSS compilers removed from the skills